projects/asn1-proto/asn1_proto_converter.cc

```diff
@@ -0,0 +1,166 @@
+#include "asn1_proto_converter.h"
+
+namespace asn1_proto {
+
+/* Returns number of bytes needed to encode num
+ * Where num is a variable-length integer with
+ * no leading zeros
+ */
+uint8_t ASN1ProtoConverter::GetNumBytes(size_t num) {
+  uint8_t shift = sizeof(num);
+  while (shift != 0) {
+    if (((num >> ((shift - 1) * 8)) & 0xFF) != 0) {
+      return shift;
+    }
+    shift--;
+  }
+  return 0;
+}
+
+/* Extracts octets from num and inserts
+ * them in encoder at the position specified
+ */
```

Comment:
I'm not sure I fully understand this comment. Does this reword mean the same?

```suggestion
// Converts |num| to a variable-length, big-endian representation and inserts
// the result into into |encoder_| at |pos|.
```

That is, function level descriptions are often better in the class file, as with https://google.github.io/styleguide/cppguide.html#Function_Comments